website/source/docs/guides/deployment.html.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: "docs"
+page_title: "Production Deployment"
+sidebar_current: "docs-guides-deployment"
+description: |-
+  Best practice approaches for Consul production deployments.
+---
+
+# Consul Production Deployment Guide
+
+As applications are migrated to dynamically provisioned infrastructure, scaling services and managing the communications between them becomes challenging. Consul’s service discovery capabilities provide the connectivity between dynamic applications. Consul also monitors the health of each node and its applications to ensure that only healthy service instances are discovered. Consul’s distributed runtime configuration store allows updates across global infrastructure.
+
+The goal of this document is to recommend best practice approaches for Consul production deployments. The guide provides recommendations on system requirements, datacenter design, networking, and performance optimizations for Consul cluster based on the latest Consul (1.0.7) release.
+
+This document assumes a basic working knowledge of Consul.
+
+## 1.0 System Requirements
```

Comment:
Given the titles are numbered should we have a table of contents? or drop the numbering? This isn't a pattern in other guides and we should probably keep it consistent.

Reply:
Fixed in df5e39ff2